docs/UserGuide.adoc

```diff
@@ -336,8 +337,8 @@ Only Gmail is accepted!
 
```

Comment:
Since you're fixing the errors for `setemail`, there's a small grammatical error in Line 332:

* `NEWEMAIL` cannot be _the_ same as `OLDEMAIL`

Reply:
I know. I will rewrite this part and remind command's part together afterwards. This is just to fix the issue.